.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -2,6 +2,18 @@
 
 ## How I Tested These Changes
 
-## Changelog [New | Bug | Docs]
+## Changelog
 
-> Replace this message with a changelog entry, or `NOCHANGELOG`
+> Replace this section with a changelog entry, or `NOCHANGELOG` if no entry is required (e.g. an internal refactor).
```

Comment:
I generally like how concise our current template is. We had a template before where I would need to delete a bunch of lines every time I posted, and I was very happy when we switched to the current one. Not 100% confident what the right balance, but would generally bias a little more towards brevity than what's here.  Also, for the educational content that we do keep here, better to use comments than indented quotes.

Reply:
Agree. How about:

```
Insert changelog entry or "NOCHANGELOG" here.
```

Also I think we could probably nuke the whole "Category" section and instead just stick the category boxes right under the "## Changelog" header?

Reply:
Other idea to consider is just letting people delete the "## Changelog" section to indicate no changelog?

Reply:
@sryza , I think deleting the section == no changelog is likely good idea in the medium term, but I've been trying to guard against old PRs without this section slipping through the cracks. So maybe in a couple weeks we can change the flow again